development/make-md-changelogs.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# Warlof Tutsimo
+# loic.leuilliot@gmail.com
+
+# The list of packages for which we want to generate markdown formatted changelog
+packages=( api console eveapi notifications services web )
+# The root repository location
+repository_directory="E:/Users/elfaus/PhpstormProjects/seatng/"
```

Comment:
replaced with `pwd` and add parameters support for both packages list and output directory :)